easybuild/easyconfigs/z/ZPAQ/ZPAQ-7.00-GCC-4.8.2.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2015 NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'MakeCp'
+
+name = 'ZPAQ'
+version = '7.00'
+
+homepage = 'http://mattmahoney.net/dc/zpaq.html'
+description = """zpaq is a free and open source (GPL v3) incremental,
+ journaling command-line archiver for Windows, Linux and Mac OS/X"""
+
+toolchain = {'name': 'GCC', 'version': '4.8.2'}
+
+sources = ['%%(namelower)s%s.zip' % ''.join(version.split('.'))]
+source_urls = ['http://mattmahoney.net/dc/']
+
+buildopts = " && pod2man zpaq.pod > zpaq.1"
```

Comment:
Either include a build dependency for `pod2man`, or remove this all together...

As is, this is a liability for breaking this build...